src/Chainweb/Pact/PactService.hs

```diff
@@ -148,6 +148,7 @@ withPactService ver cid chainwebLogger bhDb pdb sqlenv config act =
                     , _psAllowReadsInLocal = _pactAllowReadsInLocal config
                     , _psIsBatch = False
                     , _psCheckpointerDepth = 0
+                    , _psTraceLogger = logFunction chainwebLogger
```

Comment:
👍

Reply:
Later we'll just put a chainweb logger into the checkpointer as suggested by @chessai.